Mathlib/Topology/Connected/Basic.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Mario Carneiro, Yury Kudryashov
 -/
-import Mathlib.Data.Set.Image
+import Mathlib.Data.Set.Subset
```

Comment:
Adding an import that is only used after line 1000! :-) could this file be split?

Reply:
In fairness, it's a single file import (including transitive)

Reply:
#15144